Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM centos:7
+ENV JENKINS_HOME=/var/jenkins_home/
+VOLUME /var/jenkins_home/
+RUN yum -y install java
+COPY jenkins.war .
```

Comment:
you have binary data in you git repo. it is not acceptable.
please, find another way to provide the Jenkins installation into you image build context.

Reply:
Now I fixed it. So you can check it